### PR TITLE
Create And Exfiltrate A Webhook Of Discord

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+REM ###############################################################
+
+
+REM Requirements:
+REM     - Internet connection
+REM     - Discord Installed
+
+
+REM You must define the Discord server name i.e. Hak5
+DEFINE SERVER_NAME example
```
Comment (dallaswinger) [inline payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt:18]: should be `#SERVER_NAME` to match best practices

(will also need to update where it is used)

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+REM You must define your Discord webhook if you want to use this method for the exfiltration
+DEFINE DISCORD_WEBHOOK example
```
Comment (dallaswinger) [inline payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt:21]: should be `#DISCORD_WEBHOOK` to match best practices

(will also need to update where it is used)

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Settings
+
+- You must define the Discord server name i.e. Hak5
+
+    `DEFINE SERVER_NAME example`
+
+- You must define your Dropbox accessToken or modify the exfiltration modality.
+
+    `DEFINE DROPBOX_ACCESS_TOKEN example`
```
Comment (dallaswinger) [inline payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/README.md:30]: I dont see dropbox used anywhere in the actual payload?

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt (hunk 1 above)
Comment (aleff-github) [reply payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt]: renamed to `#SERVER_NAME`

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt (hunk 2 above)
Comment (aleff-github) [reply payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/payload.txt]: renamed to `#DISCORD_WEBHOOK`

File: payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/README.md (hunk 3 above)
Comment (aleff-github) [reply payloads/library/exfiltration/Create_And_Exfiltrate_A_Webhook_Of_Discord/README.md]: Resolved with the new `#DISCORD_WEBHOOK`

**Pull request merged**